Plugins/Flow.Launcher.Plugin.Explorer/Search/EnvironmentVariables.cs

```diff
@@ -44,40 +62,26 @@ internal static bool IsEnvironmentVariableSearch(string search)
 
         internal static string TranslateEnvironmentVariablePath(string environmentVariablePath)
         {
-            var envStringPaths = LoadEnvironmentStringPaths();
-            var splitSearch = environmentVariablePath.Substring(1).Split("%");
-            var exactEnvStringPath = splitSearch[0];
-
-            // if there are more than 2 % characters in the query, don't bother
-            if (splitSearch.Length == 2 && envStringPaths.ContainsKey(exactEnvStringPath))
-            {
-                var queryPartToReplace = $"%{exactEnvStringPath}%";
-                var expandedPath = envStringPaths[exactEnvStringPath];
-                // replace the %envstring% part of the query with its expanded equivalent
-                return environmentVariablePath.Replace(queryPartToReplace, expandedPath);
-            }
-
-            return environmentVariablePath;
+            return Environment.ExpandEnvironmentVariables(environmentVariablePath);
```

Comment:
maybe just update TranslateEnvironmentVariablePath call with Environment.ExpandEnvironmentVariables? Doesnt look like it needs to be wrapped inside a method.

Reply:
Sure.